### ISSUE TITLE
[Broken Website] icloud.com/mail

### ISSUE THREAD

**Issue by NSExceptional** [darkreader/darkreader#4674]
**Website**
[icloud.com/mail](https://icloud.com/mail) after signing in of course

**Problem**

Text of mail messages are not inverted

**Steps to Reproduce**

1. Sign into iCloud.com
2. Visit icloud.com/mail
3. Select a message from your inbox
4. Observe that the text of the message is not inverted

**Expected behavior**
The text of the mail message should be inverted

**Actual behavior**
It is not

**Screenshots**
<img width="50%" src="https://user-images.githubusercontent.com/8371943/103687309-b0165480-4f55-11eb-89b4-f86cf631fece.png">

**System info:**

- OS: macOS 10.15
- Browser: Chrome 87
- Darkreader Version: 4.9.26

**Comment by Gusted** [comment]
Hey @NSExceptional!

Unfortunately I cannot get into icloud mail. However I know that github notifcations uses `raw` messages. My best assumption is that the current applied color to those text is `initial` which will fall back dark-ish color. Due to the nature of initial we cannot apply a good style for it. So:
1. Does this happen with other mails(especially which don't use `raw` messages).
2. Can you check in the devtools what the current applied styles are for those texts.

Regards,
Gusted

**Comment by NSExceptional** [comment]
I think the issue is it loads the message in an `iframe`, the color is not `initial`

**Comment by Gusted** [comment]
> I think the issue is it loads the message in an `iframe`, the color is not `initial`

IFrames shouldn't be a problem for DarkReader and the browser should inject it properly. Where does the color from the messages inherit from?

**Comment by yazan1107** [comment]
yes please fix this also the reminders/notes app have white background on black backgroudn

**Comment by direrFlood** [comment]
I'm having the same issue with iCloud Notes and iCloud Remidner And iCloud beta website.

**Comment by direrFlood** [comment]
https://github.com/darkreader/darkreader/pull/10216
